### ISSUE TITLE
PHP SDK in Docker

### ISSUE THREAD

**Issue by velkymx** [ProcessMaker/bpm#1472]
Depends on https://github.com/ProcessMaker/bpm/pull/1484 and https://github.com/ProcessMaker/bpm/pull/1496
And https://github.com/ProcessMaker/executor-php/pull/2
Also the executor-php must be built and pushed to docker hub before this PR can be merged

We need to make the SDK available to script users to access the API. 

- [x] Generate SDK and docker image (see feature/1473 branch)
- [x] Generate a new bearer token every time the script runs and delete after
- [x] Pass tokens to SDK
- [x] Verify SDK works and can access the API

**Comment by nolanpro** [comment]
Currently in this branch, it takes the token from the environment variables. Taylor mentioned that he wants to generate a new token each time the script is ran then delete it after the script runs (or fails). Adding a step for this

**Comment by nolanpro** [comment]
@velkymx A few questions to wrap this one up:
- What user should run services task scripts. The scripts are defined in the BPMN and in plugins, like the email connector. They do not have any user associated.
- The migrations to associate a user to a script were not made to work for upgrades. The existing 'create_scripts_table' migration files were directly edited. Should I changed those to be their own migrations so our next release works with upgrades?
- When upgrading to the next version of pm4, some scripts will already have been created by users. What user should we set those to run as?
- Who should I talk to about publishing pm4-sdk-php repo to packagist?
- Who should handle the executor-php docker image build and can push to dockerhub?
